core/tests/webdriverio_utils/ContributorDashboardAdminPage.js

```diff
@@ -177,6 +182,21 @@ var ContributorDashboardAdminPage = function() {
       'Submit Question Right Element taking too long to appear');
     expect(await contributionRights.getText()).toBe('Allowed');
   };
+
+  this.checkstatslistelements = async function(number) {
+    await waitFor.presenceOf(statsTable, 'stats table was not visible');
```

Comment:
```suggestion
    await waitFor.presenceOf(statsTable, 'Stats table is not visible');
```

Reply:
Done. PTAL.

Reply:
I think the initial capital `S` is still not died (not sure!), can you please check and fix it?

Reply:
Done.